pkgs/development/python-modules/paddle_bfloat/default.nix

```
@@ -4,34 +4,37 @@
 , pythonOlder
 , pythonAtLeast
 , python
+, numpy
 }:
 let
   pname = "paddle_bfloat";
```

```suggestion
  pname = "paddle-bfloat";
```
It is not directly related to this PR, but changing pname before adding more dependencies to nixpkgs is a good idea.

---

I kept it with the underscore, because you can only fetch the source on pypi with the underscore.
Do you still think it's worth using the - instead ?

---

Yes, we must use `-`.
This is one of the rules specified by python.org and followed by nixpkgs.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-contributing

---

Thanks for pointing that out, I didn't know!
I've changed the file name as well, let me know if I forgot anything.